apps/condo/domains/common/utils/tables.utils.ts

```diff
@@ -124,7 +124,7 @@ export const getFilter: (
         }
         switch (argData) {
             case 'number':
-                args = search.filter(value => Number(value) && String(value).indexOf('.') === -1).map(Number)
+                args = search.filter(value => Number(value) && is32BitInteger(value) && String(value).indexOf('.') === -1).map(Number)
```

Comment:
I believe you should be able to set this up, because you may be using bigint or other types